spring-pulsar/src/test/java/org/springframework/pulsar/listener/reactive/DefaultReactivePulsarMessageListenerContainerTests.java

```diff
@@ -158,7 +160,8 @@ void containerProperties() throws Exception {
 
 		assertThat(container).extracting("pipeline", InstanceOfAssertFactories.type(ReactiveMessagePipeline.class))
 				.hasFieldOrPropertyWithValue("concurrency", 5).hasFieldOrPropertyWithValue("maxInflight", 6)
-				.hasFieldOrPropertyWithValue("handlingTimeout", Duration.ofMillis(7));
+				.hasFieldOrPropertyWithValue("handlingTimeout", Duration.ofMillis(7)).extracting("groupingFunction")
```

Comment:
Later we should consider breaking this test into smaller pieces as it is not directly clear that "setUseKeyOrderedProcessing" equates to "groupFunction == DMGF".